coalib/bearlib/languages/definitions/KornShell.py

```diff
@@ -0,0 +1,16 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class KornShell:
+    extensions = '.ksh'
+    comment_delimiters = '#',
+    multiline_comment_delimiters = {": '": "'"}
+    string_delimiters = {'"': '"', "'": "'", '`': '`'}
+    multiline_string_delimiters = string_delimiters
+    encapsulators = {'[': ']', '{': '}'}
+    keywords = [
+        'if', 'then', 'else', 'elif', 'fi', 'case', 'coproc', 'esac',
```

Comment:
Please check all keywords.
E.g. https://unix.stackexchange.com/questions/86270/how-do-you-use-the-command-coproc-in-various-shells